UdpHosts/GameServer/GameServerModule.cs

```diff
@@ -1,10 +1,10 @@
 ﻿using System;
 using System.Configuration;
 using Autofac;
-using FauFau.Formats;
 using GameServer.Physics.ZoneLoader;
 using Serilog;
 using Shared.Common;
+using SDB = FauFau.Formats.StaticDB;
```

Comment:
I find this aliasing a bit weird, is there some reason we might want it?

Reply:
No aliasing then? Encounters now use `GameServer.StaticDB.Records.customdata.Encounters` namespace to match directory structure, so `StaticDB` can't be used here